change log for aliased preview urls (#23668)
cloudflare/cloudflare-docs: src/content/changelog/workers/2025-07-23-workers-preview-urls.mdx

---

> title: Test your code changes instantly with per-branch preview deployments for Cloudflare Workers

Reviewer: ```suggestion
title: Test out code changes before shipping with per-branch preview deployments for Cloudflare Workers
```

riffing on title, "test your code changes" kind of hints towards programmatic testing, "instantly" - it's not quite instant
